recipes/perf/all/conanfile.py

```diff
@@ -0,0 +1,42 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.35.0"
+
+class Perf(ConanFile):
+    name = "perf"
+    description = "Linux profiling with performance counters"
+    topics = ("linux", "profiling")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://perf.wiki.kernel.org/index.php"
+    license = "GPL-2.0"
+    settings = "os", "compiler", "build_type", "arch"
+
+    _source_subfolder = "source_subfolder"
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    def validate(self):
+        if self._settings_build.os != "Linux":
```

Comment:
This test means that I can build `perf` on Macos for Windows with a mingw compiler.
Didn't you mean `self.settings.os != "Linux"` instead?

Reply:
I remember that I was unable to extract the linux kernel sources on a Windows NTFS partition.
Is that the problem you are checking for here?

Reply:
I meant `self.settings.os != "Linux"`